deeplearning4j-nn/src/main/java/org/deeplearning4j/nn/conf/layers/ElementWiseMultiplicationLayer.java

```diff
@@ -0,0 +1,107 @@
+package org.deeplearning4j.nn.conf.layers;
+
+import org.deeplearning4j.nn.params.ElementWiseParamInitializer;
+import lombok.*;
+import org.deeplearning4j.nn.api.Layer;
+import org.deeplearning4j.nn.api.ParamInitializer;
+import org.deeplearning4j.nn.conf.NeuralNetConfiguration;
+import org.deeplearning4j.nn.conf.inputs.InputType;
+import org.deeplearning4j.nn.conf.layers.FeedForwardLayer;
+import org.deeplearning4j.nn.conf.memory.LayerMemoryReport;
+import org.deeplearning4j.nn.conf.memory.MemoryReport;
+import org.deeplearning4j.optimize.api.IterationListener;
+import org.nd4j.linalg.api.ndarray.INDArray;
+
+import java.util.Collection;
+import java.util.Map;
+
+
+/**
+ * created by jingshu
+ */
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class ElementWiseMultiplicationLayer extends org.deeplearning4j.nn.conf.layers.FeedForwardLayer {
+
+//  ATTENTION!!! we have to add an empty constructor for custom layers otherwise we will have errors when loading the model
+    public ElementWiseMultiplicationLayer(){}
+
+    protected ElementWiseMultiplicationLayer(Builder builder){
+        super(builder);
+    }
+
+    @Override
+    public ElementWiseMultiplicationLayer clone() {
+        ElementWiseMultiplicationLayer clone = (ElementWiseMultiplicationLayer) super.clone();
+        return clone;
+    }
+
+    @Override
+    public Layer instantiate(NeuralNetConfiguration conf, Collection<IterationListener> iterationListeners, int layerIndex, INDArray layerParamsView, boolean initializeParams) {
+        if(this.nIn!=this.nOut) throw  new IllegalStateException("element wise layer must have the same input and output size!!!");
+        org.deeplearning4j.nn.layers.feedforward.elementwise.ElementWiseMultiplicationLayer ret = new org.deeplearning4j.nn.layers.feedforward.elementwise.ElementWiseMultiplicationLayer(conf);
+        ret.setListeners(iterationListeners);
+        ret.setIndex(layerIndex);
+        ret.setParamsViewArray(layerParamsView);
+        Map<String, INDArray> paramTable = initializer().init(conf, layerParamsView, initializeParams);
+        ret.setParamTable(paramTable);
+        ret.setConf(conf);
+
+        return ret;
+    }
+
+    @Override
+    public ParamInitializer initializer() {
+        return ElementWiseParamInitializer.getInstance();
+    }
+
+    /**
+     * This is a report of the estimated memory consumption for the given layer
+     *
+     * @param inputType Input type to the layer. Memory consumption is often a function of the input type
+     * @return Memory report for the layer
+     */
+    @Override
+    public LayerMemoryReport getMemoryReport(InputType inputType) {
+        InputType outputType = getOutputType(-1, inputType);
+
+        int numParams = initializer().numParams(this);
+        int updaterStateSize = (int) getIUpdater().stateSize(numParams);
+
+        int trainSizeFixed = 0;
+        int trainSizeVariable = 0;
+        if (getDropOut() > 0) {
```

Comment:
Method no longer exists. Replace condition with ```if (getIDropout() != null) {```

Reply:
ah sorry yes i was testing on another project, i test this time on deeplearning4j/master and i cannot find this method .

Reply:
for the moment i tried to build with the build stack script but it tried to test cuda which i dont have , anyway i pushed the modification you mentioned , but i dont know if it works on master